internal/service/dns_config/model_config_delegation.go

```diff
@@ -0,0 +1,155 @@
+package dns_config
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/booldefault"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/infobloxopen/bloxone-go-client/dns_config"
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type ConfigDelegationModel struct {
+	Comment           types.String `tfsdk:"comment"`
+	DelegationServers types.List   `tfsdk:"delegation_servers"`
+	Disabled          types.Bool   `tfsdk:"disabled"`
+	Fqdn              types.String `tfsdk:"fqdn"`
+	Id                types.String `tfsdk:"id"`
+	Parent            types.String `tfsdk:"parent"`
+	ProtocolFqdn      types.String `tfsdk:"protocol_fqdn"`
+	Tags              types.Map    `tfsdk:"tags"`
+	View              types.String `tfsdk:"view"`
+}
+
+var ConfigDelegationAttrTypes = map[string]attr.Type{
+	"comment":            types.StringType,
+	"delegation_servers": types.ListType{ElemType: types.ObjectType{AttrTypes: ConfigDelegationServerAttrTypes}},
+	"disabled":           types.BoolType,
+	"fqdn":               types.StringType,
+	"id":                 types.StringType,
+	"parent":             types.StringType,
+	"protocol_fqdn":      types.StringType,
+	"tags":               types.MapType{ElemType: types.StringType},
+	"view":               types.StringType,
+}
+
+var ConfigDelegationResourceSchemaAttributes = map[string]schema.Attribute{
+	"comment": schema.StringAttribute{
+		Optional:            true,
+		MarkdownDescription: "Optional. Comment for zone delegation.",
+	},
+	"delegation_servers": schema.ListNestedAttribute{
```

Comment:
can you add a validation for atleast one item required here?

Reply:
Added the validator